torch/backends/_nnapi/serializer.py

```diff
@@ -971,6 +974,69 @@ def add_flatten(self, node):
 
         self.add_operation(NNAPI_OperationCode.RESHAPE, inputs, outputs)
 
+    def add_slice(self, node):
+        assert node.inputsSize() == 5
+        assert node.outputsSize() == 1
+
+        in_id, in_oper = self.get_tensor_operand_by_jitval(node.inputsAt(0))
+        _, dim_value = self.get_constant_value(node.inputsAt(1))
+        _, start_value = self.get_constant_value(node.inputsAt(2))
+        _, stop_value = self.get_constant_value(node.inputsAt(3))
+        _, step_value = self.get_constant_value(node.inputsAt(4))
+
+        if start_value is None:
+            start_value = 0
+        if stop_value is None:
+            stop_value = sys.maxsize
+
+        if start_value < 0:
+            start_value += in_oper.shape[dim_value]
+        elif start_value == sys.maxsize:
```

Comment:
Why would this happen?

Reply:
* https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/TensorIndexing.h#L18
* https://github.com/pytorch/pytorch/blob/master/test/cpp/api/tensor_indexing.cpp#L45

None ends up translating to INDEX_MAX